conftest.py

```diff
@@ -96,3 +100,12 @@ def pytest_collection_modifyitems(config, items):
                 reason=f"Test running on shard {item_shard_index} of {num_shards}",
             )
         )
+
+
+def pytest_sessionstart():
+    if IS_IN_CI:
+        hook_script_dir = REPO_ROOT / "tests" / "scripts" / "request_hook"
+        sys.path.append(str(hook_script_dir))
+        import request_hook  # pylint: disable=import-outside-toplevel
+
+        request_hook.init()
```

Comment:
maybe print a warning here?

Reply:
the hook already logs a message any time its used + whenever it redirects a URL